### PR TITLE
Stream hot topics

### PR DESCRIPTION
**Pull request by i-like-robots**
- Re-styles hot topics for a stream
- Supports displaying 2, 3 and 4 themes
- Opinions will come back again soon

screen shot 2015-07-16 at 15 46 24
screen shot 2015-07-16 at 15 50 30
screen shot 2015-07-16 at 15 51 13

cc @andygnewman

### CODE REVIEW

File: views/partials/hot-topics.html [hunk 1]
```diff
@@ -1,33 +1,12 @@
...
-						{{#each items}}
-							<li class="hot-topics__list-item">
-								<a class="hot-topics__list-link" href="/{{id}}">{{headline}}</a>
-							</li>
-						{{/each}}
-					</ol>
-				</div>
+<div class="hot-topics">
+	<section class="hot-topics__key-topics" role="section">
+		<h2 class="stream__title">Most mentioned</h2>
```
Comment (matthew-andrews) [inline views/partials/hot-topics.html]: `Most mentioned`? is this from ux?

Comment (i-like-robots) [inline views/partials/hot-topics.html]: Design has '{{stream name}} most mentioned' but that doesn't always make sense. Will check.

**Comment by next-team** [comment]
I've spotted a [difference](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/dobi/2015-07-16/15:01-2251/failures/index.html) between your pull request and production — [see full output](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/dobi/2015-07-16/15:01-2251/successes/index.html)

**Comment by next-team** [comment]
I've spotted a [difference](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/dobi/2015-07-16/15:47-2253/failures/index.html) between your pull request and production — [see full output](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/dobi/2015-07-16/15:47-2253/successes/index.html)

**Comment by matthew-andrews** [comment]
it's a shame this stuff is still behind a flag.  would be nice to get image diffing working on it.

also i promise to fix some of the image diff bugs soon ……

**Comment by andygnewman** [comment]
+1 (and looking forward to it being opinionated soon)

**Comment by i-like-robots** [comment]
I think I'll ask for it to come out of hiding when opinions drop in.

**Pull request merged**